src/main/java/org/broadinstitute/dsde/consent/ontology/translate/utils/LoadUtils.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.dsde.consent.ontology.translate.utils;
+
+import com.github.jsonldjava.shaded.com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
+import org.broadinstitute.dsde.consent.ontology.translate.model.TermItem;
+import org.parboiled.common.FileUtils;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+public class LoadUtils {
+    public static List<TermItem> termsFromLocalResourcesJSON(@NotNull String jsonFilePath) throws Exception {
+        try {
+            final String searchTerms = FileUtils.readAllTextFromResource(jsonFilePath);
+            return new Gson().fromJson(
+                    searchTerms,
+                    new TypeToken<List<TermItem>>() {}.getType()
+            );
+        } catch (Exception e) {
+            throw new Exception("Error loading terms from json file: " + jsonFilePath, e);
```

Comment:
I don't know about this code base, but normally I wouldn't catch or throw Exception (or declare `throws Exception`) outside of test code. For `catch` I'd catch a more specific type. For `throw` I'd throw an exception type defined in my code. And unless the exception is expected to be caught by the caller, I'd use a `RuntimeException` subclass so it doesn't need to be declared as a checked exception. (Checked exceptions are fine too if that's what a code base is using.)